backend/src/Designer/EventHandlers/LanguageTextsKeyChanged/LanguageTextsIdChanged.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Events;
+using Altinn.Studio.Designer.Hubs.SyncHub;
+using Altinn.Studio.Designer.Services.Interfaces;
+using MediatR;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Altinn.Studio.Designer.EventHandlers.LanguageTextsKeyChanged;
+
+public class LanguageTextsIdChanged(
+    IHubContext<SyncHub, ISyncClient> syncHub, ITextsService textsService) : INotificationHandler<LanguageTextsKeyChangedEvent>
+{
+
+    public async Task Handle(LanguageTextsKeyChangedEvent notification, CancellationToken cancellationToken)
+    {
+        List<string> updatedFiles = await textsService.UpdateRelatedFiles(notification.EditingContext.Org, notification.EditingContext.Repo, notification.EditingContext.Developer, notification.idMutations);
+        ISyncClient syncClient = syncHub.Clients.Group(notification.EditingContext.Developer);
+        foreach (string updatedFile in updatedFiles)
+        {
+            SyncSuccess syncSuccess = new(new Source(Path.GetFileName(updatedFile), updatedFile));
+            await syncClient.FileSyncSuccess(syncSuccess);
```

Comment:
Here we should use the ` _fileSyncHandlerExecutor.ExecuteWithExceptionHandlingAndConditionalNotification` that allows us to only send the syncSuccess event if there were any changes in the layoutset.

Reply:
I'm not sure what we gain by using this function here, could you explain what you mean? I believe the file sync success is only sent for the files the changes has been done to, since `updatedFiles` is already a list of only files that has been changed.

Reply:
For consistency